manager/src/main/java/com/redhat/service/smartevents/manager/ProcessorService.java

```diff
@@ -15,9 +15,11 @@ public interface ProcessorService {
 
     Processor createProcessor(String bridgeId, String customerId, String owner, ProcessorRequest processorRequest);
 
+    Processor updateProcessor(String bridgeId, String processorId, String customerId, ProcessorRequest processorRequest);
+
     Processor createErrorHandlerProcessor(String bridgeId, String customerId, String owner, ProcessorRequest processorRequest);
 
-    Processor updateProcessor(String bridgeId, String processorId, String customerId, ProcessorRequest processorRequest);
+    Processor updateErrorHandlerProcessor(String bridgeId, String processorId, String customerId, ProcessorRequest processorRequest);
 
```

Comment:
@manstis Does it simplify our life and make things clearer if we have a `Optional<Processor> getErrorHandler(String bridgeId)` method?  It seems a little confusing to me (I'm easily confused I appreciate) that in some places we're very explicit e.g create/update Error handler, but in other's we rely on knowledge of us understanding that the list of hidden processors exists and may contain the error handler.

It existed that way before this PR, so wonder if that's a follow up change we want to make if you think it sounds sensible.

cheers,

Rob

Reply:
Done. It does avoid a duplication of code in `BridgeWorker`.